src/bff-service/src/app.controller.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Controller,
+  Get,
+  HttpStatus,
+  HttpException,
+  Res,
+  Req,
+  Inject,
+  CACHE_MANAGER,
+} from '@nestjs/common';
+import Axios, { AxiosRequestConfig } from 'axios';
+import { Cache } from 'cache-manager';
+import { Request, Response } from 'express';
+
+const cacheKey = 'products';
+
+@Controller()
+export class AppController {
+  constructor(@Inject(CACHE_MANAGER) private cacheManager: Cache) {}
+
+  @Get('*')
```

Comment:
I believe bff should not proxy only GET requests. It's better to use @All decorator here.

Reply:
Fixed.